slang-gfx.h

```diff
@@ -1232,7 +1246,8 @@ struct GraphicsPipelineStateDesc
 
 struct ComputePipelineStateDesc
 {
-    IShaderProgram*  program;
+    IShaderProgram*  program = nullptr;
+    void* d3d12RootSignatureOverride = nullptr;
```

Comment:
It would obviously be great if we eventually start working on the "extension mechanism" that will allow `gfx` to support stuff like this without it polluting each and every `Desc` type.

(Honestly, stuff like this is why I actually don't like the `*Desc` API approach, and think that there are some things OpenGL actually got right... Oh well, I am okay with being in the minority on this line of thinking...)